src/app/pages/vendor-dashboard/components/vendor-filter/vendor-filter.component.ts

```diff
@@ -1,15 +1,85 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import {COMMA, ENTER, N} from '@angular/cdk/keycodes';
+import { MatChipInputEvent } from '@angular/material/chips';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { workType } from 'src/app/types/work-type';
+import { statusTypes } from 'src/app/types/status';
+
+
 
 @Component({
   selector: 'app-vendor-filter',
   templateUrl: './vendor-filter.component.html',
   styleUrls: ['./vendor-filter.component.scss']
 })
-export class VendorFilterComponent implements OnInit {
+export class VendorFilterComponent{
+
+  addOnBlur = true;
+  readonly separatorKeysCodes = [ENTER, COMMA] as const;
+  workTypes: workType[] = [];  
 
-  constructor() { }
+  statusesTest: statusTypes[] = [
+    {name: 'in contract', state: false}, 
+    {name: 'active', state: false},
+    {name: 'I=inactive', state: false},
```

Comment:
Why is there an I=inactive?

Reply:
thats what you had in the pr from the other day so I thought it was a thing for the backend, ill remove it